documentation/Quickstart-on-how-to-write-tests.md

```diff
@@ -130,6 +137,79 @@ If we have existing recordings then the tests have been run against generated th
 > export TEST_MODE=playback
 > rushx test
 ```
+## How to push test recordings to assets repo
+We need to push test recording files to [asset repo](https://github.com/Azure/azure-sdk-assets) after testing your test cases.
+
+### Push test recording
+If your package is new without any recordings, go to the next section [New Package - No Recorded Tests](#new-package---no-recorded-tests)
+
+If you already have an assets.json file in your package root directory, go to [Existing package - Tests have been pushed before](#existing-package---tests-have-been-pushed-before).
+
+#### New Package - No recorded tests
+This section assumes that your package is new to the JS repo and that you're trying to onboard your tests with recorder, and the asset-sync workflow.
+
+Generate an `sdk/<service-folder>/<package-name>/assets.json` file by running the following command.
+
+```bash
+npx dev-tool test-proxy init
+```
+Note: If you [install `dev-tool` globally](https://github.com/Azure/azure-sdk-for-js/tree/main/common/tools/dev-tool#installation), you don't need `npx` prefix in the above command
+
+This command would generate an assets.json file with an empty tag.
+
+Example `assets.json` with an empty tag:
+
+```
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "js",
+  "TagPrefix": "js/network/arm-network",
+  "Tag": ""
+}
+```
+
+Then go to the next step to [Existing package - Tests have been pushed before](#existing-package---tests-have-been-pushed-before).
+
+#### Existing package - Tests have been pushed before
+At this point, you should have an `assets.json` file under your SDK. `sdk/<service-folder>/<package-name>/assets.json`.
+
+With asset sync enabled, there is one extra step that must be taken before you create a PR with changes to recorded tests: you must push the new recordings to the assets repo. This is done with the following command:
+
+```bash
+npx dev-tool test-proxy push
```

Comment:
Also better to mention that the tests have to be recorded using the `TEST_MODE=record`